akka-stream/src/main/resources/META-INF/native-image/com.typesafe.akka/akka-stream/reflect-config.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "name": "com.typesafe.sslconfig.ssl.DefaultHostnameVerifier",
+    "allDeclaredConstructors": true,
+    "allPublicConstructors": true
+  }
+]
```

Comment:
You can also move `akka.stream.impl.io.FileSubscriber` and `akka.stream.impl.fusing.ActorGraphInterpreter` from `akka-http` here.